database/historical_schema.sql

```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS historical_data.station_performance;
+DROP TABLE IF EXISTS historical_data.operator_performance;
+
+CREATE TABLE historical_data.station_performance(
+    station_performance_id SERIAL PRIMARY KEY,
+    station_id INT REFERENCES public.stations(station_id),
+    day TIMESTAMP NOT NULL,
+    cancellation_count INT NOT NULL,
+    delay_1m_count INT NOT NULL, 
+    delay_5m_count INT NOT NULL, 
+    avg_delay DECIMAL(4,2) NOT NULL,
+    arrival_count INT NOT NULL, 
+    common_cancel_code varchar(2) 
```

Comment:
Would it be suitable to keep this as nullable column? Or should we assign a default value to this

Reply:
So `delay_5m_count`, `avg_delay`, `cancellation_count` and `delay_5m_count` can be defaulted to zero in the case there aren't values for these when pulling data out from the short term storage